tests/sentry/incidents/endpoints/test_project_alert_rule_details.py

```diff
@@ -188,6 +193,53 @@ def test_update_snapshot(self):
                 **serialized_alert_rule,
             )
 
+    def _slack_setup_helper(self, channelName: str = None, channelID: str = None) -> Dict:
+        self.integration = Integration.objects.create(
```

Comment:
There's an existing factory `self.create_slack_integration` that you can use here.